persistence/src/commonMain/db_user/com/wire/kalium/persistence/Members.sq

```diff
@@ -51,7 +51,8 @@ SELECT
     user_message_timer,
     archived,
     archived_date_time,
-    verification_status
+    verification_status,
```

Comment:
tbh storing the mls verification status for members doesn't make sense. why we need it? it's already in the CoreCrypto we only need to fetch it. keeping it up to dated is risky.